pipes.Rmd

```diff
@@ -14,20 +14,18 @@ Avoid using the pipe when:
 
 ## Whitespace
 
-`%>%` should always have a space before it, and should usually be followed by a new line. After the first step, each line should be indented by two spaces. This structure makes it easier to add new steps (or rearrange existing steps) and harder to overlook a step.
+`|>` should always have a space before it, and should usually be followed by a new line. After the first step, each line should be indented by two spaces. This structure makes it easier to add new steps (or rearrange existing steps) and harder to overlook a step.
 
 ```{r, eval = FALSE}
 # Good
-iris %>%
-  group_by(Species) %>%
-  summarize_if(is.numeric, mean) %>%
-  ungroup() %>%
-  gather(measure, value, -Species) %>%
+iris |>
+  summarise(across(where(is.numeric), mean), .by = Species) |>
```

Comment:
Is the switch to UK spelling intentional? I think either US or UK us fine as long as consistent, though we did land on US spelling for R4DS if you think consistency across resources is good too.

Reply:
Oops.